src/js/module/layers-manager/layers-manager.js

```diff
@@ -1,7 +1,7 @@
 const LayersManager = {
```

Comment:
I think We don't need to keep modules in objects literals anymore. It's relict of previous module system where We kept classes in namespace created by module factory.

```javascript
import Item from './layers-manager/menu-item'
import Menu from './layers-manager/menu-menu'
```

Is more clear.